spec/spec_helper.rb

```diff
@@ -1,5 +1,9 @@
+require 'simplecov'
+SimpleCov.start
+
 $LOAD_PATH.unshift File.expand_path("../../", __FILE__)
 require "rspec-console"
```

Comment:
ahh i see, i thought the comment was referring to the line above, not below. i'll put this on the list.